rest-api-spec/src/main/resources/rest-api-spec/api/simulate.ingest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "simulate.ingest":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/simulate-ingest-api.html",
+      "description":"Simulates running ingest with example documents."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_ingest/_simulate",
+          "methods":[
+            "GET",
+            "POST"
+          ]
+        }
+      ]
+    },
+    "params":{
+      "index":{
+        "type":"string",
+        "description":"The default index to ingest all documents into if no index is given for a particular document"
+      },
```

Comment:
You mean instead of `_ingest/_simulate`? Or as an additional option. Right now you can pass in a different index with each document. If we made that a mandatory part of the URL we'd lose that ability (I'm not sure how useful it really is).

Reply:
I meant as an additional option, the same way it's optional for the `_bulk` API

Reply:
Makes sense. Done.